test/lib/BedrockTester.cpp

```diff
@@ -136,16 +136,7 @@ string BedrockTester::startServer(bool wait) {
             }
         }
 
-        // Convert our c++ strings to old-school C strings for exec.
-        char* cargs[args.size() + 1];
-        int count = 0;
-        for(string arg : args) {
-            char* newstr = (char*)malloc(arg.size() + 1);
-            strcpy(newstr, arg.c_str());
-            cargs[count] = newstr;
-            count++;
-        }
-        cargs[count] = 0;
+
```

Comment:
NAB extra line?

Reply:
actually there might be two extra lines https://github.com/Expensify/Bedrock/blob/e23220d159bd93052d4c9cc1e46918779b439a71/test/lib/BedrockTester.cpp#L138-L139